app/views/decidim/proposals/proposals/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<%= render partial: "decidim/shared/component_announcement" %>
+
+<% if component_settings.geocoding_enabled? %>
+  <%= dynamic_map_for proposals_data_for_map(@all_geocoded_proposals) do %>
+    <template id="marker-popup">
+      <div class="map-info__content">
+        <h3>${title}</h3>
+        <div id="bodyContent">
+          <p>{{html body}}</p>
+          <div class="map__date-address">
+            <div class="address card__extra">
+              <div class="address__icon">{{html icon}}</div>
+              <div class="address__details">
+                <span>${address}</span><br>
+              </div>
+            </div>
+          </div>
+          <div class="map-info__button">
+            <a href="${link}" class="button button--sc">
+              <%= t(".view_proposal") %>
+            </a>
+          </div>
+        </div>
+      </div>
+    </template>
+  <% end %>
+<% end %>
+<%= render partial: "voting_rules" %>
+<div class="row columns">
+  <div class="title-action">
+    <h2 id="proposals-count" class="title-action__title section-heading">
+      <%= render partial: "count" %>
+    </h2>
+    <% if current_settings.creation_enabled && current_component.participatory_space.can_participate?(current_user) &&
+      current_component.settings.try(:additional_buttons_show) %>
+      <%= action_authorized_link_to "", component_settings.additional_button_link, class: "title-action__action
+      button small" do %>
+        <%= translated_attribute(component_settings.additional_button_text) %>
+        <%= icon "plus", role: "img", "aria-hidden": true %>
+      <% end %>
+    <% elsif current_settings.creation_enabled && current_component.participatory_space.can_participate?(current_user) %>
```

Comment:
In here you are either showing the new button or the "creation button". 

The creation button is independent, it should go first. Then the "access collaborative texts" buttons (if enabled) and, finally, the custom button. All independent.

The style of the button should be using `link_to` probably